### ISSUE TITLE
Create an event fired once the grid has been fully rendered

### ISSUE THREAD

**Issue by zernyu** [SitePen/dgrid#188]
Currently, I'm using aspect.after on renderArray and using a Deferred.when on the rowsRendered deferred to run code after the dgrid has finished rendering. I've realized that this does not work with tree grids because renderArray is called for each level of nodes being rendered. It would be nice if there was a renderComplete callback or function call so we can easily run code only after the dgrid has fully completed rendering.
